docs/visualize/regionmap.asciidoc

```diff
@@ -26,10 +26,12 @@ Configure a _Terms_ aggregation. The term is the _key_ that is used to join the
 ==== Options
 
 ===== Layer Settings
-- *Vector map*: select from a list of vector maps. This list includes the maps that are hosted by the © [Elastic Maps Service](https://www.elastic.co/elastic-maps-service),
+- *Vector map*: select from a list of vector maps. This list includes the maps that are hosted by the © https://www.elastic.co/elastic-maps-service[Elastic Maps Service],
```

Comment:
I'm having issues building locally but if I recall this is the correct way to format URL's in asciidoc 😄